tests/ci/integration/postgres_patch/aws-lc-postgres.patch

```diff
@@ -1,33 +1,74 @@
+# NOTE: There are also some minor error string differences for AWS-LC
+# vs OpenSSL in "src/test/ssl/t/001_ssltests.pl" that are not within
+# this patch (AWS-LC uses all caps, OpenSSL uses lower case).
+# We use sed in "tests/ci/integration/run_postgres_integration.sh" to
+# account for these differences instead to minimize churn in this patch.
+# If we do ever get the chance to submit a patch upstream, this patch
+# should account for those error string discrepencies.
+#
+diff --git a/configure b/configure
+index 93fddd6998..6a981cc247 100755
+--- a/configure
++++ b/configure
+@@ -12805,6 +12805,18 @@ if eval test \"x\$"$as_ac_var"\" = x"yes"; then :
+ #define `$as_echo "HAVE_$ac_func" | $as_tr_cpp` 1
+ _ACEOF
+ 
++fi
++done
++
++  # Function specific to AWS-LC.
++  for ac_func in awslc_version_string
++do :
++  ac_fn_c_check_func "$LINENO" "awslc_version_string" "ac_cv_func_awslc_version_string"
++if test "x$ac_cv_func_awslc_version_string" = xyes; then :
++  cat >>confdefs.h <<_ACEOF
++#define HAVE_AWSLC_VERSION_STRING 1
++_ACEOF
++
```

Comment:
This was autogenerated via `autoconf`, which is what upstream postgres also does.